tools/pipelines/templates/build-docker-service.yml

```diff
@@ -123,6 +123,13 @@ resources:
     type: git
     name: 1ESPipelineTemplates/M365GPT
     ref: refs/tags/release
+  pipelines:
+    # Listing this pipeline as a resource makes its artifacts automatically downloaded within a deployment job.
```

Comment:
Is this true? I think it's the `templateContext.inputs` in the release job that triggers the download.

Reply:
So, in a normal deployment job, it will automatically download all artifacts from pipelines listed as resources by default. In a 1ES deployment job, you have to specify what you want to download. Nonetheless, I don't like this comment that much so its been updated in the latest commit